### PR TITLE
45-70 adjustments and speedloader tube

### PR DESCRIPTION
**Pull request by BlueHNT**
# Description

45-70 sprites looked too small and the brush gun was severely missing a speedLoader tube. So I made a western style loader tube.

Also slightly rebalances the firing speed of 45-70 weapons that have received complains. Mainly the Brush Gun and Ranger Sequoia from their 0.8 speeds.

This PR was made possible thanks to @Cojoke-dot from Wizden.

---

# TODO

- [x] Replace speedloader sprite
- [x] Add speedloader
- [x] Add crafting of the new speedLoader
- [x] make the speedLoader work

---

<details><summary><h1>Media</h1></summary>
<p>

image
image

SpeedLoader 45-70 for internal magazine working

https://github.com/Vault-Overseers/nuclear-14/assets/79374236/5bbc727d-4d9b-463d-b1e2-2d70f94357cd

</p>
</details>

### CODE REVIEW

**Comment by BlueHNT** [comment]
@Peptide90 need your assistance in making id: N14WeaponBrushGun be able to take speedLoaders. It currently loads em as a bullet for whichever reason. There are no bolt actions so I am unsure how...

File: Resources/Textures/_Nuclear14/Objects/Weapons/Guns/Ammunition/Speedloaders/45_70loader.rsi/meta.json
Comment (Pspritechologist) [inline Resources/Textures/_Nuclear14/Objects/Weapons/Guns/Ammunition/Speedloaders/45_70loader.rsi/meta.json:1]: Don't modify portions of this file that don't need changes. You seem to be lowering the indent level with no need.

File: Resources/Textures/_Nuclear14/Objects/Weapons/Guns/Ammunition/Speedloaders/45_70loadertube.rsi/meta.json
Comment (Pspritechologist) [inline Resources/Textures/_Nuclear14/Objects/Weapons/Guns/Ammunition/Speedloaders/45_70loadertube.rsi/meta.json:1]: It'd be a good idea for the indents here to match the other files. Four spaces is standard for indentation, but whatever editor you're using is automatically adjusting to two.

File: Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml
Comment (Pspritechologist) [inline Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:1]: Your editor is also stripping line endings automatically. This isn't a huge deal since they shouldn't be there to begin with, and this is an N14 file, but not that it's not happening to Upstream files when you commit because that causes merge conflicts all the time.

File: Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 1]
```diff
@@ -125,10 +125,15 @@
     sprite: _Nuclear14/Objects/Weapons/Guns/Snipers/brushgun.rsi
   - type: Item
     sprite: _Nuclear14/Objects/Weapons/Guns/Snipers/brushgun.rsi
+  - type: Gun
+    fireRate: 3
+    soundGunshot:
+      path: /Audio/_Nuclear14/Weapons/Firing/brushgunfire.ogg
```
Comment (Pspritechologist) [inline Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:131]: This change doesn't seem related to anything talked about in the pr Readme. Make sure it's intended, and if it is, to talk about what it's actually doing and why.

File: Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml [hunk 2]
```diff
@@ -110,6 +110,8 @@
     sprite: _Nuclear14/Objects/Weapons/Guns/Revolvers/ranger_sequoia.rsi
   - type: Clothing
     sprite: _Nuclear14/Objects/Weapons/Guns/Revolvers/ranger_sequoia.rsi
+  - type: Gun
+    fireRate: 2
```
Comment (Pspritechologist) [inline Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml:114]: This also seems totally unrelated.

File: Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+﻿- files: ["war_never_changes.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "BigIron github and it's sources"
```
Comment (Pspritechologist) [inline Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml:3]:
```suggestion
  copyright: "BigIron github and its sources"
```
This is quite an unclear Copyright. You should clarify what 'BigIron' is (I.E, a repo, a person, etc.) and what 'its sources' if referring to, and potentially include an actual link somewhere if applicable.

File: Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+﻿- files: ["war_never_changes.ogg"]
```
Comment (Pspritechologist) [inline Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml:1]: This seems to be attributing a file that isn't added in this pr??

File: Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Ammunition/Speedloaders/45-70.yml [hunk 5]
```diff
@@ -38,3 +38,24 @@
...
+    layers:
+      - state: base
+        map: [ "enum.GunVisualLayers.Base" ]
+      - state: base-10
+        map: [ "enum.GunVisualLayers.Mag" ]
+  - type: MagazineVisuals
+    magState: base
+    steps: 11
+    zeroVisible: false
+  - type: Appearance
```
Comment (Pspritechologist) [inline Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Ammunition/Speedloaders/45-70.yml:61]:
```suggestion

- type: entity
  id: SpeedLoader45-70Tube
  name: "speed loader tube (45-70 gov)"
  parent: N14BaseSpeedLoader45-70
  components:
  - type: BallisticAmmoProvider
    proto: N14Cartridge45-70
    capacity: 10
  - type: Sprite
    sprite: _Nuclear14/Objects/Weapons/Guns/Ammunition/Speedloaders/45_70loadertube.rsi
  - type: MagazineVisuals
    steps: 11
    zeroVisible: false
```
These should not be needed.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

File: Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml (hunk 3 above)
Comment (BlueHNT) [reply Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml:3]: The copyright is a local open sourced github that closed down. I may aswell not put any copyright there.

File: Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml (hunk 4 above)
Comment (BlueHNT) [reply Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml:1]: Seems I forgot to change this.

File: Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml
Comment (BlueHNT) [reply Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:1]: I do not make changes to the upstream controlled places and let the VSC strip ends with the N14 stuff I work on.

File: Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml (hunk 3 above)
Comment (Pspritechologist) [reply Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml:3]: That's not how copyright works.

File: Resources/Textures/_Nuclear14/Objects/Weapons/Guns/Ammunition/Speedloaders/45_70loader.rsi/meta.json [hunk 6]
```diff
@@ -23,7 +23,7 @@
             "name": "base-3"
         },
         {
-            "name": "base-4"
+        "name": "base-4"
```
Comment (Pspritechologist) [inline Resources/Textures/_Nuclear14/Objects/Weapons/Guns/Ammunition/Speedloaders/45_70loader.rsi/meta.json:26]:
```suggestion
            "name": "base-4"
```

**Review by Pspritechologist — CHANGES_REQUESTED** [review]
